frontend/src/metabase/admin/upsells/components/UpsellBigCard.tsx

```diff
@@ -52,16 +64,27 @@ export const _UpsellBigCard: React.FC<UpsellBigCardProps> = ({
           <Title order={1} lh={1} mb="sm">
             {title}
           </Title>
-          <Text lh="lg" mb="lg">
+          <Text lh="xl" mb="lg">
             {children}
           </Text>
-          <ExternalLink
-            className={S.UpsellCTALink}
-            href={url}
-            onClickCapture={() => trackUpsellClicked({ source, campaign })}
-          >
-            {buttonText}
-          </ExternalLink>
+          {buttonLink ? (
+            <ExternalLink
+              className={S.UpsellCTALink}
+              href={url}
+              onClickCapture={() => trackUpsellClicked({ source, campaign })}
+            >
+              {buttonText}
+            </ExternalLink>
+          ) : (
+            <Box
```

Comment:
Couldn't use `Button` with any of the variants because the styles compete. Polymorphic `Box` does not have these problems.